test/python/test_parallel.py

```diff
@@ -37,14 +39,16 @@ def test_parallel(self):
     def test_parallel_progressbar(self):
         """Test parallel_map with progress bar"""
         TextProgressBar()
-        ans = parallel_map(_parfunc, list(range(10)))
+        with patch('sys.stdout', new=StringIO()) as _:  # Mute the progress bar itself
```

Comment:
I had to do this in #737 to get things running on windows with the different test runner. Do you think it makes sense to see if we call write() here like I did there? Not sure it provides much value but having the mock lets us verify we're writing what we want to stdout (or writing at all).

Reply:
I liked your way better. So now it's part of this PR in 988afbf.